src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLinear.cs

```diff
@@ -19,27 +19,41 @@ namespace Microsoft.ML.Trainers.Online
 
     public abstract class OnlineLinearArguments : LearnerInputBaseWithLabel
     {
+        /// <summary>
+        /// Number of training iterations through the data.
+        /// </summary>
         [Argument(ArgumentType.AtMostOnce, HelpText = "Number of iterations", ShortName = "iter", SortOrder = 50)]
         [TGUI(Label = "Number of Iterations", Description = "Number of training iterations through data", SuggestedSweeps = "1,10,100")]
         [TlcModule.SweepableLongParamAttribute("NumIterations", 1, 100, stepSize: 10, isLogScale: true)]
-        public int NumIterations = OnlineDefaultArgs.NumIterations;
```

Comment:
>NumIterations [](start = 19, length = 13)

Set the Name attribute to be the old name everytime you change one of them; for maml backwards compat.

Reply:
i added it to shortname. did i do it right?

---
In reply to: [256252619](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256252619) [](ancestors = 256252619)